yarn.lock

```diff
@@ -260,14 +260,6 @@
     "@babel/helper-plugin-utils" "^7.8.3"
     "@babel/plugin-syntax-jsx" "^7.8.3"
 
```

Comment:
why did yarn.lock change? we didn't change any `package.json` files.

It only deletes entries, so it doesn't matter much (no new code being injected), but it's curious. Does it have to do with the `asset-manifest.json` file changing?

Reply:
🤷‍♂ Yarn.